lib/ramble/ramble/application.py

```diff
@@ -64,6 +65,7 @@ class ApplicationBase(object, metaclass=ApplicationMeta):
     def __init__(self, file_path):
         super().__init__()
 
+        self._pipelines = ['setup', 'analyze', 'archive', 'mirror']
```

Comment:
Should we pull this out and validate calls to `run_pipeline` against it? I feel like we currently have the different pipelines strongly implied in different places and adding a new pipeline might be a bit odd/error prone right now?

Reply:
Should be better now. Let me know if you have concerns with my latest update. Thanks for the suggestion!